.github/workflows/cifuzz.yml

```diff
@@ -0,0 +1,35 @@
+name: CIFuzz
+on: [pull_request]
+jobs:
```

Comment:
Can we restrict the permission to contents read?
```suggestion
name: CIFuzz

on: [pull_request]

permissions:
  contents: read

jobs:
```

Reply:
I'm not sure either, could we get CIFuzz to run in this PR?

Reply:
done.